### PR TITLE
Linear Search completed

### LINKED ISSUE #7: Search Algorithms : Linear Search
**Is your Algorithm request related to a problem? Please describe.**
I want to add linear Search algorithm

**Describe the solution you'd like**
Linear search algorithm

**Describe alternatives you've considered**
Linear search

### PR DESCRIPTION
**Pull request by Souph**
### Pull request title
- [ ] Clearly and concisely describes what it does
- [ ] Refer to the issue that it solves, if available

### Pull request description

- [ ] Describes the main changes that come with the pull request
- [ ] Any relevant additional information is provided

resolves #7

### CODE REVIEW

File: src/main/java/xyz/subho/algorithms/search/LinearSearch.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        //Linearsearch function called which will return the index of the key if found, else -1
+        //Array along with the key passed as parameter
+        int index = LinearSearch(Array, key);
+        
+        if(index!=-1)
+            System.out.print(key + " found at index " + index);
+        else 
+            System.out.print(key + " not found in array");
+    }
+}
```
Comment (ohbus) [inline src/main/java/xyz/subho/algorithms/search/LinearSearch.java:36]: Add a new line at the end of the file.

**Review by ohbus — CHANGES_REQUESTED** [review]
Please run `./mvnw spotless:apply` to format the code.

**Comment by Souph** [comment]
Linear search update 2.0

**Review by ohbus — APPROVED** [review]

**Comment by ohbus** [comment]
@all-contributors please add @Souph for code

**Comment by ohbus** [comment]
@all-contributors please add @Souph for code

**Pull request merged**